samples/container/kafka-function/src/main/java/com/contoso/kafka/TriggerFunction.java

```diff
@@ -0,0 +1,53 @@
+package com.contoso.kafka;
+
+import java.util.*;
+import com.microsoft.azure.functions.annotation.*;
+import com.microsoft.azure.functions.*;
+
+import com.microsoft.azure.functions.kafka.annotation.*;
+import com.microsoft.azure.functions.kafka.*;
+
+/**
+ * Azure Functions with HTTP Trigger.
+ */
+public class TriggerFunction {
+    /**
+     * This function consume KafkaEvents on the localhost. Change the topic, brokerList, and consumerGroup to fit your enviornment.
+     * The function is trigged one for each KafkaEvent
+     * @param kafkaEventData
+     * @param context
+     */
+    // @FunctionName("KafkaTrigger-Java")
```

Comment:
Do you mind remove the comments?

Reply:
I have two functions `KafkaTrigger-Java` and `KafkaTrigger-Java-Many` The first one is a sample for accessing local clusters with Cardinality.ONE. The second one is a sample for accessing Confluent  Cloud with Cardinality.MANY.  For the other samples, I comment out the second one, since a lot of getting started customers don't have a Confluence Cloud. So I keep a sample for local Kafka Cluster and comment out the sample for Confluence Cloud.  This sample is for Docker, It aims to deploy to the Function App. That means it can't access the local cluster. So I comment out the local cluster sample and enable the Confluent Cloud sample. Should I remove the local cluster sample instead?